charts/istioconfig/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: v2
+name: terra-istio-config
```

Comment:
The name of the chart should be the name of the folder. `istioconfig` is better since we don't want dashes in our chart names. See naming doc below:
https://github.com/broadinstitute/terra-helmfile/blob/master/docs/NAMING.md